src/astro/sql/operators/dataframe.py

```diff
@@ -114,13 +127,17 @@ def execute(self, context: Dict) -> Union[Table, pd.DataFrame]:
             self.database = self.database or first_table.metadata.database  # type: ignore
             self.schema = self.schema or first_table.metadata.schema  # type: ignore
         self.op_args = load_op_arg_table_into_dataframe(
-            self.op_args, self.python_callable, self.identifiers_as_lower
+            self.op_args, self.python_callable, self.columns_names_capitalization
         )
         self.op_kwargs = load_op_kwarg_table_into_dataframe(
-            self.op_kwargs, self.python_callable, self.identifiers_as_lower
+            self.op_kwargs, self.python_callable, self.columns_names_capitalization
         )
 
         pandas_dataframe = self.python_callable(*self.op_args, **self.op_kwargs)
+        pandas_dataframe = convert_dataframe_col_case(
```

Comment:
I'd expect the column capitalization logic to be handled within the database class (`load_pandas_dataframe_to_table`) since the motivation to introduce this complexity was the limitations of databases.

Reply:
@tatiana aren't we overpopulating the database class?

Reply:
As per discussion with @tatiana we agreed that this functionality spans across the lib, wherever we are creating the dataframe and this param introduces a way to control cols case. Hence resolving this comment.